### ISSUE TITLE
LESS_TERMCAP standout color settings not working

### ISSUE THREAD

**Issue by danielclucas** [walles/moar#114]
I just recently switched over to `moar` from `less` and it seems that `moar` doesn't respect the LESS_TERMCAP standout color settings as `less` does. My standout color settings are set by:
```bash
export LESS_TERMCAP_so=$'\E[01;38;2;187;194;207;48;2;56;122;167m'
```
This should produce highlighting with a light blue background and white foreground text. However, `moar` appears to be only inverting the color of the text and background for standout mode.

Per https://github.com/walles/moar/issues/14 `moar` should respect LESS_TERMCAP settings in the same way as `less`.

**Comment by walles** [comment]
I tried this with `less`, and the only thing that turned blue was the prompt at the bottom of the page.

Can you give an example / screenshot / repro case / something to demonstrate what you want to turn blue with this setting?

Since I don't use the `LESS_TERMCAP` settings myself I need some sort of test case to see that I'd be fixing the right thing.

**Comment by walles** [comment]
And search hits, I saw that now as well.

**Comment by danielclucas** [comment]
Yes, no problem. Here is a photo of what I see upon opening man pages.
![screenshot02](https://user-images.githubusercontent.com/49924063/210132272-9aa4d69a-9ab2-496c-8546-fddf9414bd23.png)
 Top is moar and bottom is less. You can see that the standout (highlighting) settings are applied to the bottom notification/banner in less but not moar. 

Here is a photo of what I see upon searching:
![screenshot01](https://user-images.githubusercontent.com/49924063/210132308-5390e8d6-35d4-4370-af04-c336a4dfea0e.png)
Again, top is moar and bottom is less. You can see that search items are highlighted with colors according to termcap settings in less, but are simply inverting colors in moar.

**Comment by walles** [comment]
Cool, so it's search hits and the bottom status bar that should get the `_so` color.

**Comment by walles** [comment]
Released 1.11.3 with this change in it. If you're on Homebrew, here's the release status: https://github.com/Homebrew/homebrew-core/pull/119468

Please let me know how it works for you!

**Comment by danielclucas** [comment]
@walles It works perfectly! Thanks for making that update. I've now switched over to using `moar` entirely in place of `less`.
